views/islandora_usage_stats.views.inc

```diff
@@ -67,6 +67,75 @@ function islandora_usage_stats_views_data() {
       ),
     ),
   );
+  $data['islandora_usage_stats_datastreams'] = array(
+    'table' => array(
+      'group' => t('Islandora Usage Stats'),
+      'base' => array(
+        'field' => 'id',
+        'title' => t('Islandora Usage Stats Datastreams'),
+        'help' => t('Datastream information.'),
```

Comment:
This title makes sense, except its not clear that it is just a table of the datastreams downloaded tied to the PID of the Object. Not sure how to clarify what the purpose is here.